editions/2023/fa/0x01-about-owasp.md

```diff
@@ -0,0 +1,66 @@
+# <div dir="rtl">درباره OWASP </div>
```

Comment:
I don't see any noticeable difference with/without the `<div>`:

![image](https://github.com/user-attachments/assets/8e13df62-686a-4315-be02-41421ab610f1)

Please carefully review the document removing non-required `<div dir="rtl">` throughout it.